### DISCUSSION TITLE
My Paladin buys 10 antidote potions before doing andy and immediately drinks them all?

### DISCUSSION THREAD

**Discussion by zylo-** [isid0re/SoloLeveling#136] [General]
Just watching my Paladin and he's doing andy in nightmare (he's lvl 80), and he goes to Akara before Andy, buys 10 antidote potions, and immediately drinks all 10 before going back to the tp

Why is this?

EDIT: Nevermind, apparently the durations stack when you drink multiple potions! Amazing.

**Comment by zylo-** [comment]
i just watched my sorc do the same thing, buys 8 stamina potions in act 2 and immediately drinks them all

**Comment by isid0re** [comment]
Yes the antidote and thawing potions are bought and stacked before certain key areas.

Stamina potions are bought if the bot doesnt have teleport in acts 2 and 3 as a time saving measure for the larger act sizes.
